### ISSUE TITLE
You still require VPN to get the access

### ISSUE THREAD

**Issue by hinsiu1228** [Henryyy-Hung/ChatGPT-Playground-By-API#1]

**Comment by Henryyy-Hung** [comment]
Yes, but once you obtain the API key, you no longer need to access the official website of OpenAI, and you no longer need a VPN. I created this to avoid spending half a minute turning on my VPN every time I want to use the GPT chat instead of completely get rid of VPN.

**Comment by Henryyy-Hung** [comment]
I understand your question now. Some areas, like China, block OpenAI by using a firewall. Therefore, I have added a proxy API to avoid being banned by the firewall. You may try it now.
